lint.go

```diff
@@ -52,12 +52,12 @@ func (p *Problem) String() string {
 	return p.Text
 }
 
-type byPosition []Problem
+type ByPosition []Problem
```

Comment:
I need to sort the problems in golint/golint.go. I do the linting package by package adding problems to one slice. At the end, this slice has to be sorted.

Any suggestion for a name? "ProblemsByPosition"?